initials.c

```diff
@@ -0,0 +1,13 @@
+#include <stdio.h>
```

Comment:
Great job! The only thing I'd add is a comment at the top with your name and function of your code.